app/models/program.rb

```diff
@@ -15,4 +15,5 @@ class Program < ActiveRecord::Base
   }.freeze
 
   validates :facility_code, presence: true
+  belongs_to :insitutions
```

Comment:
This should be `belongs_to :institution` I think

Reply:
Nice catch. That definitely was a typo.